app/src/main/java/org/oppia/android/app/topic/lessons/ChapterSummaryViewModel.kt

```diff
@@ -23,14 +24,22 @@ class ChapterSummaryViewModel(
   }
 
   fun computeChapterPlayStateIconContentDescription(): String {
-    return if (chapterPlayState == ChapterPlayState.COMPLETED) {
-      resourceHandler.getStringInLocaleWithWrapping(
-        R.string.chapter_completed, (index + 1).toString(), chapterTitle
-      )
-    } else {
-      resourceHandler.getStringInLocaleWithWrapping(
-        R.string.chapter_in_progress, (index + 1).toString(), chapterTitle
-      )
+    return when (chapterPlayState) {
+      ChapterPlayState.COMPLETED -> {
+        resourceHandler.getStringInLocaleWithWrapping(
+          R.string.chapter_completed, (index + 1).toString(), chapterTitle
+        )
+      }
+      ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES -> {
```

Comment:
Please add a test for this follow-up change & add two screenshots to the PR description: one showing the new test failing when your fix is temporarily commented out/removed, and a second showing it passing. This is generally good practice when fixing targeted issues like this.

Reply:
I have added a test for the ideal situation (where there is a previous chapter title). I haven't been able to add a test for the other condition since I am unable to replicate it in test.

Reply:
Thanks! The other case is, theoretically, impossible. There is a way to force it to happen in tests, but I think it makes sense just to ignore it. We might be able to eliminate the codepath entirely with future refactors of the domain structures, but I think it's fine to leave it as-is untested.